docs/api/api.yml

```diff
@@ -386,29 +386,32 @@ paths:
     post:
       tags:
       - directory
-      summary: Adds a file or a subdirectory.
+      summary: Adds one or more files.
```

Comment:
so how would you add a subdirectory, without a specific file?

Reply:
There is no way to just create a subdirectory, except by sending a ZIP file that contains one empty directory.